tests/unit/job/test_query.py

```diff
@@ -244,6 +244,7 @@ def test_ctor_defaults(self):
         self.assertIsNone(job.destination_encryption_configuration)
         self.assertIsNone(job.range_partitioning)
         self.assertIsNone(job.time_partitioning)
+        self.assertIsNone(job.timeout_ms) //// 
```

Comment:
Don't need trailing `////`

Reply:
Resolved, it was an accidental comment, won't happen again.